src/pages/wallet/Nft.vue

```diff
@@ -60,134 +44,104 @@ const transferNFT = () => {
-      <div class="flex h-16 px-4">
-        <div class="flex-1 flex items-center mr-auto">
-          <router-link to="/wallet/home">
-            <img class="block h-4 w-auto" :src="ControllerModule.isDarkMode ? SolanaLightLogoURL : SolanaLogoURL" alt="Solana Logo" />
-          </router-link>
-        </div>
-        <div v-if="selectedAddress && user.verifierId" class="flex flex-3">
-          <div class="hidden md:-my-px md:mx-auto md:flex md:space-x-0">
-            <router-link
-              v-for="(value, key) in tabs"
-              :key="key"
-              :to="`/wallet/${value.route}`"
-              :class="[
-                key === 'nfts'
-                  ? 'border-app-primary-500 text-app-primary-500'
-                  : 'border-transparent text-gray-500 hover:border-gray-300 dark:hover:border-white hover:text-gray-700 dark:hover:text-white',
-                'inline-flex items-center px-4 pt-1 border-b-2 text-sm font-medium',
-              ]"
-              aria-current="page"
-              >{{ t(value.name) }}</router-link
-            >
-          </div>
-        </div>
-        <div v-if="selectedAddress && user.verifierId" class="hidden md:flex items-center flex-1 ml-auto justify-end">
-          <LanguageSelector class="mr-2" /><AccountMenu :user="user"
-            ><AccountMenuList :user="user" :selected-address="selectedAddress" @on-logout="logout"
-          /></AccountMenu>
-        </div>
-        <div v-if="selectedAddress && user.verifierId" class="ml-6 flex md:hidden items-center">
-          <AccountMenuMobile><AccountMenuList :user="user" :selected-address="selectedAddress" @on-logout="logout" /></AccountMenuMobile>
-        </div>
-      </div>
-    </nav>
-    <main v-if="nftMetaData" class="flex-1 relative">
-      <div class="h-[380px] w-full absolute top-0 left-0 flex justify-center items-center overflow-hidden">
-        <img
-          alt="NFT"
-          :src="nftMetaData.offChainMetaData?.image || FallbackNft"
-          class="object-cover w-full h-full"
-          @error="setFallbackImg($event.target, FallbackNft)"
-        />
-        <div class="h-full w-full bg-gray-800 backdrop-blur-lg bg-opacity-30 absolute top-0 left-0"></div>
-      </div>
-      <!-- content -->
-      <div class="px-4 gt-xs:px-12 md:px-12 py-8 pb-6 relative flex flex-col items-center w-full h-full justify-center md:flex-row md:items-start">
-        <div class="flex flex-col">
-          <div class="flex justify-between px-2 py-4 w-full">
-            <div class="cursor-pointer flex items-center" @click="goBack" @keydown="goBack">
-              <ChevronLeftIcon class="text-app-text-dark-400 h-4 w-4 mr-2" />
-              <span class="text-app-text-dark-400 text-base font-light">Back</span>
-            </div>
-            <DotsHorizontalIcon class="h-6 w-6 text-app-text-dark-400 cursor-pointer" />
-          </div>
+  <WalletTabs tab="nfts" :show-header="false">
```

Comment:
![image](https://user-images.githubusercontent.com/94340415/156099416-61090699-6c23-4c2b-8984-f2a1a197213d.png)

NFT page should use entire screen width and height. `WalletTabs` add a padding-x that we don't require, we also don't need the extra space at bottom for mobile navbar for when the user is not logged in.

Reply:
Fixed